tests/integration/tests/auth.test.ts

```diff
@@ -18,61 +18,75 @@ describe("basic auth", async () => {
 	});
 });
 
-describe("scope/record auth", async () => {
+describe("scope auth", async () => {
 	const surreal = await createSurreal();
 	const version = await surreal.version();
+	if (!version.startsWith("surrealdb-1")) return;
 
-	if (version.startsWith("surrealdb-1")) {
+	beforeAll(async () => {
 		await surreal.query(/* surql */ `
-			DEFINE TABLE user PERMISSIONS FOR select WHERE id = $auth;
-			DEFINE SCOPE user
-				SIGNUP ( CREATE type::thing('user', $id) )
-				SIGNIN ( SELECT * FROM type::thing('user', $id) );
-		`);
-
-		test("scope signup", async () => {
-			const signup = await surreal.signup({
-				scope: "user",
-				id: 123,
-			});
-
-			expect(typeof signup).toBe("string");
+    		DEFINE TABLE user PERMISSIONS FOR select WHERE id = $auth;
+    		DEFINE SCOPE user
+    			SIGNUP ( CREATE type::thing('user', $id) )
+    			SIGNIN ( SELECT * FROM type::thing('user', $id) );
+    	`);
+	});
```

Comment:
I would use before each to make sure the tests are isolated

Reply:
It already uses a single database instance though, and the tests themselves won't affect schema. It should not be an issue this way